tests/run_test.sh

```diff
@@ -0,0 +1,30 @@
+## Set up vagrant-libvirt
+# getent group libvirt
+# groups ${USER}
+# sudo gpasswd -a ${USER} libvirt
+# newgrp libvirt
+## Fix for Silverblue
+# sudo restorecon -Rv /etc/libvirt # -> Relabeled /etc/libvirt from system_u:object_r:virt_etc_rw_t:s0 to system_u:object_r:virt_etc_t:s0
+
+cd atomic
+
+vagrant up
+# Use the failing check to test red boot status behaviour
+# vagrant ssh -c 'ln -s /home/vagrant/sync/10_failing_check.sh /etc/greenboot/check/required.d/10_failing_check.sh'
+
+# Workaround for https://github.com/fedora-selinux/selinux-policy/issues/242
+vagrant ssh -c 'sudo semanage fcontext -a -t pam_var_run_t "/var/run/motd.d(/.*)?"; \
```

Comment:
Minor thing, but this regex should be updated to the corrected form `/var/run/motd\.d(/.*)?` (https://github.com/fedora-selinux/selinux-policy/pull/244#discussion_r250683367).